### PR TITLE
Move colspec and rename arguments

### PR DESCRIPTION
**Pull request by eliascarv**
This PR moves the colspec file to src and renames `cols` to `names` for consistency with the rest of the code.

### CODE REVIEW

File: src/transforms.jl [hunk 1]
```diff
@@ -211,7 +211,7 @@ end
 # IMPLEMENTATIONS
 # ----------------
 
-include("transforms/colspec.jl")
+include("colspec.jl")
```
Comment (juliohm) [inline src/transforms.jl:214]: This include should go in the main TableTransforms.jl file. This list of includes here is for implementations of the Transform trait.

**Review by juliohm — CHANGES_REQUESTED** [review]

File: src/transforms.jl (hunk 1 above)
Comment (eliascarv) [reply src/transforms.jl:214]: ok

Comment (eliascarv) [reply src/transforms.jl]: Made in [933754d](https://github.com/JuliaML/TableTransforms.jl/pull/38/commits/933754de0e18e04bafc02872229966abf01cbfa8)

**Review by juliohm — APPROVED** [review]

**Comment by juliohm** [comment]
Fix #36

**Pull request merged**